bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageWebTargets.java

```diff
@@ -43,15 +44,18 @@ public ControllerVariables getControllerVariables() throws OpenGarageCommunicati
         return ControllerVariables.parse(response);
     }
 
-    public void setControllerVariables(boolean request) throws OpenGarageCommunicationException {
+    public void setControllerVariables(OpenGarageCommand request) throws OpenGarageCommunicationException {
         logger.debug("Received request: {}", request);
-        if (request) {
-            logger.debug("Received request to open door");
-            String queryParams = "&open=1";
-            invoke(changeControllerVariablesUri, queryParams);
-        } else {
-            logger.debug("Received request to close door");
-            String queryParams = "&close=1";
+        String queryParams = null;
+        switch (request) {
+            case OPEN:
+              queryParams = "&open=1";
+            case CLOSE:
+              queryParams = "&close=1";
+            case CLICK:
+              queryParams = "&click=1";
```

Comment:
You forgot to add break statements.
```suggestion
              queryParams = "&open=1";
              break;
            case CLOSE:
              queryParams = "&close=1";
              break;
            case CLICK:
              queryParams = "&click=1";
              break;
```

Reply:
Fixed in last commit